src/main/java/ch/jalu/configme/properties/EnumSetProperty.java

```diff
@@ -0,0 +1,31 @@
+package ch.jalu.configme.properties;
+
+import ch.jalu.configme.properties.types.EnumSetPropertyType;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Introduce an EnumSetProperty that goes along with EnumSetPropertyType.
+ *
+ * @param <E> enum
+ */
+public class EnumSetProperty<E extends Enum<E>> extends SetProperty<Set<E>> {
+
+    public EnumSetProperty(String path, Class<E> enumClass, EnumSet<E> defaultValue) {
+        super(path, new EnumSetPropertyType(enumClass), defaultValue);
```

Comment:
(Nothing for you to do with this comment, just thinking out loud:

I need to put some thought into whether SetProperty is properly defined as `Property<Set<E>>` because it clashes with more specific types like `EnumSetPropertyType`, which is typed to return `EnumSet<E>`—not `Set<E>`. On the one hand, I don't think any other specific set type will come up unless we'll want to have `TreeSet` in the future... and I do want an enum set property to be of type `SetProperty` so that someone picking up property classes can easily do `instanceof` checks for all sets/lists for example without needing to list a dozen.
On the other hand, this is theoretically not correct and may come to bite me later because a `Set<E>` (like `HashSet`) can be supplied in methods where a PropertyType typed as `EnumSet` might expect exclusively `EnumSet`. The way the types are defined in the `Property` classes, an EnumSetPropertyType implementation would be right to assume that there can only be an `EnumSet` coming in as argument into a method like `toExportValue`. We muddy this contract here from PropertyType to `Property` impl by losing that `EnumSet` information.)